doc/connecting.md

```diff
@@ -45,6 +49,41 @@ and before disposing the stream, you can hold the connection open as long as the
 await rpc.Completion;
 ```
 
+## Configuring/customizing the protocol
+
+To alter the protocol in any way from the defaults, use the constructor directly instead.
```

Comment:
I would say "use the JsonRpc constructor directly, instead of using the Attach method"

Reply:
Done.